packages/angular/test/base/src/app/lazy/form-controls/range/range.component.html

```diff
@@ -11,6 +11,7 @@
           <div slot="label">Range</div>
         </ion-range>
       </ion-item>
+      <p>Value: {{ form.controls['range'].value }}</p>
```

Comment:
While talking to Sean about testing this, he had the idea to manually dispatch an `ionInput` event and check if the value changes, but that doesn't work here because the value is pulled directly from the component, which naturally doesn't update.